orb.tmpl.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         default: ""
     docker:
       - image: "{{docker_image}}"
+    resource_class: xlarge
```

Comment:
Does setting this mean it's not configurable by the user? Or was it never configurable? If so let's add a parameter for it.

Reply:
It wasn't configurable. I debated adding a parameter but thought it wasn't worth it